students/esokada/lesson08/circle.py

```diff
@@ -0,0 +1,69 @@
+import math
+
+from functools import total_ordering
+
+
+@total_ordering
+class Circle(object):
+
+    def __init__(self, radius):
+        self.radius = radius
+
+    @classmethod
+    def from_diameter(cls, diameter):
+        radius = diameter / 2
```

Comment:
this can be removed since you're not using `radius`

Reply:
Thanks for catching this! Must have been left over, I don't know what I was thinking.